developers/contributing_how_tos.md

```diff
@@ -0,0 +1,41 @@
+# How to contribute a napari How-To
+
+This should be a sentence (or brief paragraph) about what this how-to will achieve and how it helps address a common problem/task/need.
+
+## Prerequisites
+
+- this template
+- a list of what the user needs to prepare to be ready to start this how-to guide 
+- links to any existing napari.org tutorials or how-to guides that can help the user fulfill these prerequisites
+- the level of python/napari knowledge required to follow this how-to
+    - try to be specific about what skills are needed e.g. 
+    'connecting callbacks to layer events' or 'using matplotlib to produce plots'
+- plugins that should be installed
+- python packages that should be installed (don't list napari dependencies)
+
+***
+
+## How-to steps
+
+Headings in this section are yours to decide on. You should split your how-to guide into easy to follow, sequential steps to achieve the stated goal. Remember that your how-to may be used as a reference, so your headings should be descriptive and allow users to get a quick overview of the document, and which steps are relevant to them.
+
+## Include pictures
+
+You should include pictures/videos, particularly when describing interactions with the viewer. If an action can be performed both through the viewer and through code, you should include both options.
+
+## How-to vs. tutorial
+
+A how-to is a step by step guide designed to answer a specific "How do I..." question. This means the user already knows **what** they want to do, just not how. A how-to does not explain concepts or engage in discussion, and should prioritise practical usability. In comparison, a tutorial is a broader exploration, or guided tour of what is available. It might include detailed explanations and discussion, and is usually aimed at users who don't yet know what can be done.
```

Comment:
I think it would be clearer to focus on what to do, or what makes a good how-to document.

It becomes more confusing for readers if there's a lot of:
* Sentences saying "don't do X or Y" (don't think of unicorn! ...now that's all you can think about, right?)
* Sentences trying to clarify the difference between a how-to and a tutorial

... you know, I think what I'm saying here is that you are inserting too much explanation content into your own how-to guide

Reply:
> ... you know, I think what I'm saying here is that you are inserting too much explanation content into your own how-to guide

Yes, I too had this feeling. I will reword to focus more on the how-to rather than how-not-to, and we can maybe work on disambiguation of tutorials and how-to's elsewhere

Reply:
+1, great suggestion @GenevieveBuckley - I think this section reads very nicely already if you just remove the 'don't do this' parts:

```suggestion

## How-to vs. tutorial

A how-to is a step by step guide designed to answer a specific "How do I..." question. This means the user already knows **what** they want to do, just not how.
```

Reply:
I think this doesn't need to be removed, actually. I quite like the idea of making sure people understand this from the beginning. I'd suggest:

```suggestion
A how-to is a step by step guide designed to answer a specific "How do I..." question. This means the user already knows **what** they want to do, just not how. A how-to does not explain concepts or engage in discussion, and should prioritise practical usability.
```

This sentence:
```
In comparison, a tutorial is a broader exploration, or guided tour of what is available. It might include detailed explanations and discussion, and is usually aimed at users who don't yet know what can be done.
```
could go into the tutorial exemplar/guide.